src/DateInput.js

```diff
@@ -16,10 +19,13 @@ export default class DateInput extends Component {
 
     state = { value: null, typeValue: null };
 
+    autoCorrectedDatePipe = createAutoCorrectedDatePipe('dd-mm-yyyy');
```

Comment:
Seems like the date format is now hardcoded while the component used to support a format property.

Completely mapping luxon style date formats to the ones text-mask-addons supports might be a bit farfetched but maybe we should just only support auto complete for a fixed set of common date formats? (Think `dd-mm-yyyy`, `mm-dd-yyyy`, and `yyyy-mm-dd` for example)

Reply:
Yes, I will use the format property instead.